### ISSUE TITLE
Angular 9 ContentChildren breaking change - Nested steps are not detected 

### ISSUE THREAD

**Issue by TomaszCz** [madoar/angular-archwizard#265]
Hi,

After recent adjustments in how the @ContentChildren - descendants: false flag works 
[details](https://github.com/angular/angular/issues/14769#issuecomment-586726706)

It is now impossible to use wizard with steps nested in any elements.
Reproduction repository:
[repository](https://github.com/TomaszCz/angular-archwizard-repro/commits/master)
If you try this repo, you will see, that ContentChildren has not detected any step directives:
![image](https://user-images.githubusercontent.com/20865719/74932879-22845180-53e3-11ea-9bbe-9a7f56b2d0de.png)

It breaks because of extra div:
```html
<aw-wizard>
    <div>
        <aw-wizard-step stepTitle="Title of step 1">
            Content of Step 1
        </aw-wizard-step>
    </div>
</aw-wizard>
```
To solve this, it would be good to add { descendants : true } in this line:
https://github.com/madoar/angular-archwizard/blob/47f91cf8f7174d69233b5ccd3db8daa988f18fbd/src/lib/components/wizard.component.ts#L58

Pull request proposition : #266

**Comment by madoar** [comment]
Thank you for reporting this issue and also for presenting a fix for it.

My initially considered application scenario was to directly wrap the wizard steps (e.g. `aw-wizard-step`) in an `aw-wizard` environment. This also allows for an easier understanding of how the wizard is setup (i.e. which steps does it consist of and how are they ordered). Because of my initial application case it is difficult for me to evaluate whether this is a bug or whether this can be perhaps considered as a feature.

To allow me to better understand the situation and to do a better evaluation, can you provide me a short use-case where it makes sense to wrap the wizard steps in a container that is then added to the wizard environment? In addition it would also be useful to hear additional benefits this approach provides you.

**Comment by TomaszCz** [comment]
Appreciate quick response.

Of course, my main argument is that this fix makes adjustment to the recent breaking change in angular. 

In terms of example, in one of my projects, I am sharing buttons logic between steps so that the steps don't need to implement them. Also, i am using bootstrap panel styling.

Essential html ( removed many things for brevity ):
```html
<aw-wizard>
    <div class="panel-heading">{{ wizardTitle | translate }}</div>
    <div class="panel panel-default">
        <div class="panel-body">
            <my-component-step awWizardStep></my-component-step>
            <my-component-step awWizardStep></my-component-step>
            <my-component-step awWizardStep></my-component-step>
            ...
            <my-component-step awWizardStep></my-component-step>
        </div>
        <div class="panel-footer">
            <button awPreviousStep></button>
            <button awNextStep></button>
        </div>
    </div>
</aw-wizard>
```

Since the version 3.0.0 such approach worked fine with no issues.

**Comment by madoar** [comment]
So essentially the two parts:

```html
        <div class="panel-heading">{{ wizardTitle | translate }}</div>
```

and 

```html
        <div class="panel-footer">
            <button awPreviousStep></button>
            <button awNextStep></button>
        </div>
```

Are replicated into every `my-component-step` element? I.e. similar to:

```html
<my-component-step awWizardStep>
   {{ wizardTitle | translate }}
   ...
   <button awPreviousStep></button>
   <button awNextStep></button>
</my-component-step>
```

**Comment by TomaszCz** [comment]
The buttons are not replicated. I verified the rendered DOM and it looks pretty much similar to what I pasted in my example. 
Also, please take a look at the bootstrap part. It is not easy to make panel class working without some CSS tweaks.

The whole point of creating the buttons once, and use them across all steps - which may be added / removed dynamically - is to follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle.

**Comment by madoar** [comment]
This is indeed an interesting approach. Thank you for explaining the idea behind it!

It would be nice to have a demo for this approach in the [demo repository](https://github.com/madoar/angular-archwizard-demo). If you are interested I would be happy to add a simple example for this to the demo repository!

**Comment by madoar** [comment]
This issue has been fixed in #266
